xml/System.Data.Common/DbDataReader.xml

```diff
@@ -1033,18 +1032,10 @@ private static void GetDataTypes(String connectionString) {
         <Parameter Name="ordinal" Type="System.Int32" Index="0" FrameworkAlternate="dotnet-uwp-10.0;netcore-1.0;netcore-1.1;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;netstandard-2.0;netstandard-2.1;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0" />
       </Parameters>
       <Docs>
-        <typeparam name="T">Synchronously gets the value of the specified column as a type.</typeparam>
-        <param name="ordinal">The column to be retrieved.</param>
-        <summary>Synchronously gets the value of the specified column as a type.</summary>
-        <returns>The column to be retrieved.</returns>
-        <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- For more information, see [SqlClient Streaming Support](~/docs/framework/data/adonet/sqlclient-streaming-support.md).  
-  
- ]]></format>
-        </remarks>
+        <typeparam name="T">The type of the value to be returned.</typeparam>
+        <param name="ordinal">The zero-based column ordinal.</param>
+        <summary>Synchronously gets the value of the specified column as the requested type.</summary>
```

Comment:
May want to get rid of word "synchronously" in these methods.

Reply:
I wonder what convention is in other BCL APIs where both sync and async methods exist...